src/api/app/queries/event_subscriptions_finder.rb

```diff
@@ -3,10 +3,11 @@ def initialize(relation = EventSubscription.all)
     @relation = relation
   end
 
-  def for_scm_channel_with_token(event_type:, event_package:)
+  def for_scm_channel_with_token(event_type:, event_package:, event_request:)
```

Comment:
You require both keywords now but you only ever will get one.

Reply:
true, I adapted it to decide which query to do based on the object class